src/main/twirl/uk/gov/hmrc/play/views/layouts/betaBanner.scala.html

```diff
@@ -19,15 +19,15 @@
         showFeedbackLinkInBetaBanner: Boolean = true,
         noBorder: Boolean = false)
 
-<div class="beta-banner@if(noBorder) { beta-banner--borderless}">
-    <p><strong class="phase-tag">BETA</strong><span>This is a new service.
-        @if( showFeedbackLinkInBetaBanner ) {
-            Help us improve it - send your
-            @defining(if (userLoggedIn) betaFeedbackUrl else betaFeedbackUnauthenticatedUrl) { linkUrl =>
+<div class="beta-banner @if(noBorder) {beta-banner--borderless}">
+    <p>
+        <strong class="phase-tag">BETA</strong>
+        <span>This is a new service @if(showFeedbackLinkInBetaBanner) {
```

Comment:
The `<span>` has associated styles in assets-frontend https://github.com/hmrc/assets-frontend/blob/0551d318fe24a4ddb1fca4f45ff64a6ece265b11/assets/govuk_elements/govuk/public/sass/design-patterns/_alpha-beta.scss#L29-L38